packages/core/test/financial-templates/PricelessPositionManager.js

```diff
@@ -795,13 +804,33 @@ contract("PricelessPositionManager", function(accounts) {
       expectedSponsorCollateralSynthetic
     );
 
-    // Check return value.
+    // Redeem and settleExpired should give the same result as just settleExpired.
     const settleExpired = pricelessPositionManager.settleExpired;
-    const redeemedAmount = await settleExpired.call({ from: sponsor });
-    assert.equal(redeemedAmount.toString(), expectedTotalSponsorCollateralReturned.toString());
 
-    // Execute the settlement and check balances.
+    // Get the amount as if we only settleExpired.
+    const settleExpiredOnlyAmount = await settleExpired.call({ from: sponsor });
+
+    // Partially redeem and partially settleExpired.
+    const partialRedemptionAmount = await pricelessPositionManager.redeem.call(
```

Comment:
Can you add a test case where you call settleExpired first and then redeem? I think ordering matters because based on the resolved price, redeeming 1 token would unlock more or less collateral than settleExpiring 1 token

Reply:
Is that possible? Won't settleExpired clear your position? Or do you want to double check that the redeem will fail?

Reply:
> Won't settleExpired clear your position?

Oh you're right